src/web/nextui/src/app/api/eval/job/[id]/route.ts

```diff
@@ -1,47 +0,0 @@
-import invariant from 'tiny-invariant';
-import { NextRequest, NextResponse } from 'next/server';
-import { cookies } from 'next/headers';
-import { createRouteHandlerClient } from '@supabase/auth-helpers-nextjs';
-
-import { getJob, getResult, SupabaseJobStatus, SupabaseEvaluationJob } from '@/database';
-import { IS_RUNNING_LOCALLY, USE_SUPABASE } from '@/constants';
-import evalJobs from '../evalJobsStore';
-
-export const dynamic = IS_RUNNING_LOCALLY ? 'auto' : 'force-dynamic';
-
-export async function GET(req: NextRequest, { params }: { params: { id: string } }) {
-  if (IS_RUNNING_LOCALLY) {
-    return NextResponse.json({ error: 'Not implemented' });
-  }
-  if (USE_SUPABASE) {
-    const supabase = createRouteHandlerClient({ cookies });
-    const { id } = params;
-    let job: SupabaseEvaluationJob | undefined;
-    try {
-      job = await getJob(supabase, id);
-    } catch (err) {
-      console.error(err);
-      return NextResponse.json({ error: 'Job not found' }, { status: 404 });
-    }
-
-    invariant(job, 'Job not found');
-
-    if (job.status === SupabaseJobStatus.COMPLETE) {
-      const result = await getResult(supabase, id);
-      return NextResponse.json({ status: 'complete', result });
-    } else {
-      return NextResponse.json({ status: 'in-progress', progress: job.progress, total: job.total });
-    }
-  } else {
-    const id = params.id;
```

Comment:
FYI, this route and others like it are used to run evals on the hosted app.promptfoo.dev site.  It may be safe to remove this functionality if we want to simplify the hosted site to shared results only and lean into self-hosting for actual eval runs.

Reply:
Does the hosted app use the root-directory Dockerfile?  Either way, I'll do another pass to confirm that these changes don't break self-hosting.

I like that app.promptfoo.dev has full functionality: it allows potential-users to evaluate promptfoo without the friction of setting up a local environment.  To this end, I spoke with @mldangelo about adding 1-click deployments (e.g. to Vercel, Fly.io), but lack the context necessary to weigh their prioritization.